src/components/EmptyPhraseMessage.spec.js

```diff
@@ -0,0 +1,29 @@
+import { render, screen } from '@testing-library/react';
+import { MemoryRouter as Router } from 'react-router-dom';
+import EmptyPhraseMessage from './EmptyPhraseMessage.js';
+
+describe('EmptyPhraseMessage', () => {
+  it('renders a title, a message text and link', () => {
+    render(
+      <Router>
+        <EmptyPhraseMessage
+          titleText="Upps...da fehlt noch was!"
+          emptyPhrasetext="Gehe zurück und markiere deine Lieblingsprüche einfach durch anklicken auf das Herzsymbol!"
```

Comment:
I am not really the greatest in grammar, I am pretty sure `durch anklicken auf das Herzsymbol` is incorrect. Try `durch Anklicken des Herzsybols` or `durch Klicken auf das Herzsymbol`.